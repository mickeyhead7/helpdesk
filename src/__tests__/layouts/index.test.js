import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Layout from '../../layouts';

const search = 'foo';
const content = () => <p>Some content...</p>;

test('Header', () => {
  const component = shallow(<Layout search={search}>{content}</Layout>);

  expect(toJson(component)).toMatchSnapshot();
});
