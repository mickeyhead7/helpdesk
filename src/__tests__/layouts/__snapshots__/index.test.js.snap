// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
    meta={
      Array [
        Object {
          "content": "Sample",
          "name": "description",
        },
        Object {
          "content": "sample, something",
          "name": "keywords",
        },
      ]
    }
    title="Helpdesk"
  />
  <Header
    search=""
  />
  <main
    className="mainContent"
  >
    <p>
      Some content...
    </p>
  </main>
</div>
`;
